Content.Shared/Inventory/VirtualItem/SharedVirtualItemSystem.cs

```diff
@@ -76,18 +76,40 @@ private void OnBeforeRangedInteract(Entity<VirtualItemComponent> ent, ref Before
     /// </summary>
     /// <param name="blockingEnt">The entity we will make a virtual entity copy of</param>
     /// <param name="user">The entity that we want to insert the virtual entity</param>
-    public bool TrySpawnVirtualItemInHand(EntityUid blockingEnt, EntityUid user)
+    public bool TrySpawnVirtualItemInHand(EntityUid blockingEnt, EntityUid user, bool dropOthers = false)
```

Comment:
Param doc

Reply:
Fixed in [`96d0ca9` (#27975)](https://github.com/space-wizards/space-station-14/pull/27975/commits/96d0ca91ac9c8429bbb92883fe2f404e841ef28d)